open-budget/app/scripts/searchbar.coffee

```diff
@@ -110,14 +110,14 @@ class BudgetPartitionLayoutView extends Backbone.View
         @cls = (d,suffix="") => window.changeClass( d.value, d.value*(d.o+100)/100.0 ) + suffix
 
         onSuccess = (root) =>
+            if @root
```

Comment:
Yes it should be 'if root'

It protected from cases that only happen because the local database is still incomplete